.github/workflows/studio_release.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  repository_dispatch:
+    types: [studio-release]
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  release:
+    concurrency:
+      group: release
+    runs-on: ubuntu-latest
+    env:
+      STUDIO_VERSION: ${{ github.event.client_payload.version }}
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          for expression in \
+            '.version |= (split(".") | .[-1] |= ((. type = "!!int") + 1) | join("."))' \
+            '.appVersion = env(STUDIO_VERSION)'
+          do
+            go run github.com/mikefarah/yq/v4@v4.34.1 eval --inplace "$expression" charts/studio/Chart.yaml
+          done
+      - uses: iterative/setup-cml@v1
+      - run: >
+          cml pr create
+          --{title,body,message}="Studio: bump to $STUDIO_VERSION"
+          --branch="bump-studio-$STUDIO_VERSION"
+          --squash
+          .
+        env:
+          GITHUB_TOKEN: ${{ secrets.OLIVAW_PAT }}
+      - run: >
+          gh api repos/${{ github.repository }}/pulls/$(gh pr view --{json=,jq=.}number)/reviews
+          --field event=APPROVE
```

Comment:
@0x2b3bfa0 let's temporary comment out this and let approving be manual.

Also, I wish I will fix the workflow on tomorrow to make a regression testing to perform checks if the new variable was not introduced or something ;-)

Reply:
> @0x2b3bfa0 let's temporary comment out this and let approving be manual.


:heavy_plus_sign:

Reply:
Resolved with https://github.com/iterative/helm-charts/pull/115/commits/e202f3faad10d6d65e3a0c3c7f871bf859359837